core/src/main/java/com/scalar/db/storage/common/checker/ColumnChecker.java

```diff
@@ -16,11 +16,13 @@
 @NotThreadSafe
 class ColumnChecker implements ValueVisitor {
   private final TableMetadata tableMetadata;
+  private final boolean notNullOrEmpty;
   private String name;
   private boolean isValid;
 
-  public ColumnChecker(TableMetadata tableMetadata) {
+  public ColumnChecker(TableMetadata tableMetadata, boolean notNullOrEmpty) {
```

Comment:
```suggestion
  public ColumnChecker(TableMetadata tableMetadata, boolean neitherNullNorEmpty) {
```
since it can be read as `not null` or `empty`.

Reply:
Thank you for taking a look at this! I modified it in the latest commit. Thanks.